tests/test_runner.js

```diff
@@ -447,4 +449,55 @@ describe('Dialogue', () => {
 
     expect(run.next().done).to.be.true;
   });
+
+  it('Can insert values into message using variable storage', () => {
+    runner.load(formattingYarnData);
+    const run = runner.run('Start');
+
+    const value = run.next().value;
+
+    expect(value.text).to.equal("Hello Adventurer! who's your day going ?");
+  });
+
+  it('Can insert plural dependant of number', () => {
+    runner.load(formattingYarnData);
+
+    let run = runner.run('NoMoney');
+    let value = run.next().value;
+    expect(value.text).to.equal('You have no gold, what a shame...');
+    run = runner.run('LowMoney');
+    value = run.next().value;
+    expect(value.text).to.equal('You have one piece inside your pocket.');
+    run = runner.run('HighMoney');
+    value = run.next().value;
+    expect(value.text).to.equal('You have 175 gold pieces!');
```

Comment:
ah nice! You did add the tests :clap: :tada:  Thank you! Wow we might be able to merge this today!

Reply:
i don't added the tests for time and duration yet, i'll do that later on